sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_models.py

```diff
@@ -4178,23 +4178,29 @@ def from_dict(cls, data):
             if data.get("innererror") else None
         )
 
-def _get_children(source_element, search_elements, cross_page=False):
+def _get_children(source_element, search_element, cross_page=False):
     result = []
 
     # search for elements across pages if cross_page is set to True
     if cross_page:
-        for elem in search_elements:
-            for region in source_element.bounding_regions:
-                for element in _get_element_list(_get_page(source_element._parent.pages, region.page_number), elem):
-                    if _in_span(element, source_element.spans):
-                        result.append(element)
+        for region in source_element.bounding_regions:
+            for element in _get_element_list(
+                _get_page(
+                    source_element._parent.pages,
+                    region.page_number,
+                    ),
+                search_element,
+            ):
+                if _in_span(element, source_element.spans):
+                    result.append(element)
+                if _beyond_span(element, source_element.spans):
```

Comment:
do we need to sort the spans before doing a check like this? I thought that was Paul's latest feedback.

Reply:
I dont remember seeing that feedback O.o. Where was it? Or should I ping Paul??

Reply:
It was in the FR SDK sync chat:

> The spans should be ordered by reading order, which ideally has incrementing indices. Unfortunately, this is not true in general since we don't yet include selection marks in the overall reading order computation (they are listed at the end). So though the assumption about the index ordering should be true in the future, I would recommend that we sort the spans in the implementation to avoid relying on such behavior.

Reply:
Ah yes, I had only taken that as relevant for selection marks...but hmmm you're right. I can do it since we dont have any assurances for the order although so far I've seen ordered spans. This will add some more computation :/ ok will think about this and add it.